### PR TITLE
Issue#33/upload resume cl

### PR DESCRIPTION
**Pull request by Kunal22shah**
- fixed issues with getResume and getCoverLetter ( they now function as expected)
- wrote test to cover the whole resumeAndCoverletter controller
-I also updated the createUser in the user controller to return an id, since i need it for my test.
- checked to see that they pass:
image

-postman response (getResume): 
image

The only part that I need to do now for my user story is to implement all of this with the edit user profile frontend.

### CODE REVIEW

File: src/server/controllers/usersController.js [hunk 1]
```diff
@@ -86,39 +86,39 @@ const updateUser = async(req, res) => {};
...
+
+    const users = await User.find({
+        firstname: { $regex: firstname, $options: "i" },
+    }).then((users) => {
+        if (users) {
+            res.status(200).json(users);
+        } else {
+            return res.status(400).json({ message: "No user exists with this name" });
+        }
+    });
```
Comment (imran1289-ah) [inline src/server/controllers/usersController.js:100]: i have a controller that has the same name "search" i believe, will this make any conflicts? If possible can you change this name

**Review by imran1289-ah — APPROVED** [review]
everything seems fine for me ! Good work !!

**Comment by imran1289-ah** [comment]
For your test, how does it work? does it upload a dummy text or an actual file

**Review by Aboudik1452 — APPROVED** [review]
Looks good!
Please resolve the conflict that is mentioned below

**Pull request merged**